expertsystem/schemas/xml/particle.json

```diff
@@ -0,0 +1,112 @@
+{
+  "$id": "https://github.com/ComPWA/expertsystem/blob/master/schemas/xml/particle.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$ref": "#definitions/Particle",
+  "definitions": {
+    "Particle": {
+      "type": "object",
+      "required": ["Name", "Pid", "Parameter", "QuantumNumber"],
+      "additionalProperties": false,
+      "properties": {
+        "Name": { "type": "string" },
+        "Pid": {
+          "anyOf": [
+            { "type": "string", "pattern": "^([+-]?[0-9]+)$" },
+            { "type": "integer" }
+          ]
+        },
+        "Parameter": { "$ref": "#/definitions/Parameter" },
+        "QuantumNumber": { "$ref": "#/definitions/QuantumNumbers" },
+        "DecayInfo": {
+          "type": "object",
+          "required": ["Parameter"],
+          "additionalProperties": false,
+          "properties": {
+            "Type": { "enum": ["relativisticBreitWigner", "flatte"] },
+            "FormFactor": { "$ref": "#/definitions/FormFactor" },
+            "Parameter": { "$ref": "#/definitions/ParameterList" }
+          }
+        }
+      }
+    },
+    "FormFactor": {
+      "type": "object",
+      "required": ["Type"],
+      "additionalProperties": false,
+      "properties": {
+        "Type": { "enum": ["BlattWeisskopf", "1"] }
+      }
+    },
+    "Parameter": {
+      "type": "object",
+      "required": ["Type", "Name", "Value"],
+      "additionalProperties": false,
+      "properties": {
+        "Type": { "type": "string" },
+        "Name": { "type": "string", "pattern": "^[A-Za-z]*_.*$" },
+        "Value": { "$ref": "#/definitions/XmlScalar" },
+        "Error": { "$ref": "#/definitions/XmlScalar" },
+        "Min": { "$ref": "#/definitions/XmlScalar" },
+        "Max": { "$ref": "#/definitions/XmlScalar" },
+        "Fix": { "$ref": "#/definitions/XmlBoolean" },
+        "ParticleA": { "type": "string" },
+        "ParticleB": { "type": "string" }
```

Comment:
It's just there, in `particle_list.xml`, so I put a check on that as well (otherwise the IDE validation would fail). Up to you what to do with it: it should not be there. But then again, dynamics info should not be in `particle_list.xml` in the first place. See also https://github.com/ComPWA/expertsystem/pull/110#issuecomment-656055278.

Reply:
I see. I guess its some strange flatte syntax... so lets just leave that in for now.